UPGRADE-7.4.md

```diff
@@ -44,6 +44,11 @@ HttpFoundation
 
  * Deprecate using `Request::sendHeaders()` after headers have already been sent; use a `StreamedResponse` instead
 
+Routing
+-------
+
+ * Deprecate `Route::getEnv()` and `Route::setEnv()` in favor of the `Route::getEnvs()` and `Route::setEnvs()` methods.
```

Comment:
On reviewing, I realize that there are multiple Route classes and that this can be confusing. Let's add the namespace.

```suggestion
 * Deprecate `getEnv()` and `setEnv()` methods of the `Symfony\Component\Routing\Attribute\Route` class in favor of the plurialized `getEnvs()` and `setEnvs()` methods. They get and return an array of env names.
```